packages/jest/src/matchers.ts

```diff
@@ -5,6 +5,8 @@ import type { MatchFilter } from './types';
 
 type Arg = [{ [key: string]: string }, MatchFilter?];
 
+/** With the babel plugin in `increaseSpecificity:true`, classes will be appended with this. */
+const INCREASE_SPECIFICITY_SELECTOR = ':not(#\\9)';
```

Comment:
nit: maybe this could be moved to `@compiled/utils`, as well as the `INCREASE_SPECIFICITY_SELECTOR` definition in `packages/css/src/plugins/increase-specificity.ts`, considering that this `:not(#\\9)` string is pretty esoteric

not too important though

Reply:
Ah, I didn't know we had a utils; done in 87df1354

Reply:
Nevermind, hit a TS circular reference issue, so I did end up copy and pasting: https://github.com/atlassian-labs/compiled/pull/1656#discussion_r1555243279 😅